spring-cloud-generator/scripts/generate-library-list.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+set -ex
```

Comment:
`-e` will reduce the log output and help us isolate the error by exiting earlier.
`-o` will save us from a local run due to lack of output in the runner log.